string/string.mbt

```diff
@@ -11,3 +11,190 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+pub fn iter(self : String, f : (Char) -> Unit) {
+  for i = 0; i < self.length(); i = i + 1 {
+    f(self[i])
+  }
+}
+
+test "iter" {
```

Comment:
we are planning to support multi-platform, so we are a bit cautious adding lots of String/Char functions

Reply:
Okay, this pull request will return to the draft.